src/any_agent/frameworks/any_agent.py

```diff
@@ -16,6 +16,7 @@
     TracingConfig,
 )
 from any_agent.logging import logger
+from any_agent.serving.server import serve_a2a
```

Comment:
We should make this import also lazy, like we handle `_get_a2a_server`

Reply:
Along the lines of https://github.com/mozilla-ai/any-agent/blob/e7873751c12305d54bb906d398e2e04c3a7a55ff/src/any_agent/serving/server.py#L20 maybe?